tests/test_all.py

```diff
@@ -172,6 +174,28 @@ def test_zypper_verify_passes(container_per_test: ContainerData) -> None:
     )
 
 
+@pytest.mark.skipif(
+    DOCKER_SELECTED, reason="Trivy-Scan images only with podman"
+)
+@pytest.mark.skipif(
+    LOCALHOST.system_info.arch not in ("x86_64",),
+    reason=f"{LOCALHOST.system_info.arch} is not supported to run trivy",
+)
+def test_trivy_image_scan(host, auto_container):
+    """Check that trivy image is able to scan the container."""
+
+    trivy_container = "docker.io/aquasec/trivy:latest"
+    host.run_expect(
+        [0],
+        (
+            f"podman run -v trivy:/root {trivy_container} image "
+            f"{auto_container.image_url_or_id} "
```

Comment:
will trivy work if `auto_container.image_url_or_id` is the hash of a locally built image?

Reply:
Actually, since we support running containers from local storage, there is **no** guaranteed way to obtain the url of a container :-(

Reply:
That's fine in a way, we're running it just in the defined list of URLs, not arbitrary ones. Also I could add a skipif it doesn't look like an url..